dat.h

```diff
@@ -17,6 +17,10 @@ typedef uint64_t      uint64;
 #define int64_t  do_not_use_int64_t
 #define uint64_t do_not_use_uint64_t
 
+/* WARNING: definitions of ms, job and tube are confusing.
+   They introduce types that are pointers. Probably,
+   they should be renamed to indicate their nature. */
+
```

Comment:
I vote for ms_ptr, job_ptr and so on. btw, how about TODO instead of WARNING?

Reply:
I agree it would be a good TODO to change these types. But I wouldn't introduce names like anything_ptr. Instead we can make these three types look like the others (e.g. Conn, Heap, File, etc), which are typedefs for struct types of the same names. I got this style from some Plan 9 programs, and as you can see I never got all the way through converting everything.

Reply:
I was meaning to make them look like the other type aliases, hiding pointer does not make it easy to read. Converting this into TODO

Reply:
Filed #458